scenarios/aws/microVMs/microvms/files/download-vm-images.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
```

Comment:
This is adding python as a dependency on the metal instance, correct? I don't think we currently require python.

Reply:
We don't require it, but AFAIK Python is always available, a lot of system tools already rewuure it. In this case I didn't have to add any dependency to the metal instance AMI, I also used only standard libraries to avoid extra dependencies. 

I tried to use Bash at first, but parsing the images to download wasn't trivial and the code would have ended up more complex and fragile.